### DISCUSSION TITLE
match _error and index pages on function pattern

### DISCUSSION THREAD

**Discussion by franciscorubin** [vercel/next.js#15932] [Help]
I'm trying to match the 404 and index pages with a function pattern on the vercel config, but none of the paths are working.

```
{
    "functions": {
        "pages/**/*.js": {
            "excludeFiles": "public/**/*"
        },
        "404.js": {
            "excludeFiles": "public/**/*"
        },
        "_error.js": {
            "excludeFiles": "public/**/*"
        }
    }
}
```

My goal is to ignore the public folder on the serverless functions, as they're causing my build to fail with the following message:

```
Error: The Serverless Function "_error" is 165.1mb which exceeds the maximum size limit of 50mb. Learn More: https://vercel.link/serverless-function-size
```

**Comment by samcx** [comment]
Hello,

In production, files in public will not start with the path public. They will instead be at your root path (/).

**Reply by franciscorubin** [reply]
That doesn't seem to apply to file matching in vercel.json
